src/styles/blocks/product-card.scss

```
@@ -0,0 +1,82 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  margin-top: 50px;
```

```suggestion
  margin-top: 50px;
```
Можливо краще весь блок з класом **product-card** огорнути в новий блок і добавити цей відступ.
На твій розгляд

---

згоден тільки давай додамо клас який буде елементом, зробим мікс

---

Этот новый класс должен быть элементом product-field или product-card?